bayes_opt/target_space.py

```diff
@@ -136,23 +205,44 @@ def mask(self):
 
         return mask
 
+
     def params_to_array(self, params):
-        try:
-            assert set(params) == set(self.keys)
-        except AssertionError:
+        """Convert a dict representation of parameters into an array version.
+
+        Parameters
+        ----------
+        params : dict
+            a single point, with len(x) == self.dim.
+
+        Returns
+        -------
+        np.ndarray
+            Representation of the parameters as dictionary.
```

Comment:
```suggestion
            Representation of the parameters as an array.
```
I think the previous version was a typo

Reply:
Good catch, thanks!